docker/1.11/Dockerfile.cpu

```diff
@@ -18,7 +18,7 @@ RUN \
 
 # cython, falcon, gunicorn
 RUN \
-    pip3 install cython falcon gunicorn gevent requests
+    pip3 install cython falcon gunicorn gevent requests tensorflow-serving-api
```

Comment:
this introduces a huge set of dependencies (all of tf, numpy, grpcio and more)

Reply:
Yes. Installing tensorflow-serving-api brings lots of dependencies. I'll do --no-dependencies and install tensorflow, grpcio and protobuf separately.